src/util/outputNameTemplate.js

```diff
@@ -65,15 +65,23 @@ export function getOutSegError({ fileNames, filePath, outputDir, safeOutputFileN
 // This is used as a fallback and so it has to always generate unique file names
 // eslint-disable-next-line no-template-curly-in-string
 export const defaultOutSegTemplate = '${FILENAME}-${CUT_FROM}-${CUT_TO}${SEG_SUFFIX}${EXT}';
-
+let currentTimestamp = Date.now();
 function interpolateSegmentFileName({ template, inputFileNameWithoutExt, segSuffix, ext, segNum, segLabel, cutFrom, cutTo, tags }) {
   const compiled = lodashTemplate(template);
+
+  if (segSuffix) {
+    if (segNum > 1) {
+      currentTimestamp += 1;
+    } 
+  }
+
   const data = {
     FILENAME: inputFileNameWithoutExt,
     SEG_SUFFIX: segSuffix,
     EXT: ext,
     SEG_NUM: segNum,
     SEG_LABEL: segLabel,
+    TIMESTAMP: currentTimestamp,
```

Comment:
Maybe we should call it epoch_millis instead

Reply:
I named it EPOCH_MS

Reply:
I think this change is fine, as long as there is a good way to explain the purpose of this button to the user. It should be stressed that the "timestamp / epoch_ms" is not necessarily a 100% accurate timestamp but just a unique identifier we made for each segment. Not sure on the best way to convey that though.

Reply:
Maybe we should instead remove the above `currentTimestamp + i` code. Then if people want to ensure a unique identifier per file, they can use `${EPOCH_MS}_${SEG_NUM}`. I think that's more clean, what do you think?

Reply:
In general, I think it's just a matter of how we can clearly convey these instructions to the user. Maybe we should update the error messages for duplicate filenames to also say something about `${EPOCH_MS}` and `${SEG_NUM}`, if we remove the `currentTimestamp + i` code. Or we can also try tooltip popups for whenever the user hover cursor over a button.

If it's clear to the user how to use these buttons, then I think either approach is fine.

After thinking more about it though, I agree that having the user switch to `${EPOCH_MS}_${SEG_NUM}` if they are making multiple segments might be more clear in the output file names. For instance, if the user trims 5 segments at once it may be easier to track those 5 segments together if the `EPOCH_MS` value stays the same.